airflow/www/api/experimental/endpoints.py

```diff
@@ -184,13 +184,11 @@ def task_info(dag_id, task_id):
     return jsonify(fields)
 
 
-# ToDo: Shouldn't this be a PUT method?
-@api_experimental.route('/dags/<string:dag_id>/paused/<string:paused>', methods=['GET'])
+@api_experimental.route('/dags/<string:dag_id>/pause/<string:pause_state>', methods=['POST'])
```

Comment:
```suggestion
@api_experimental.route('/dags/<string:dag_id>/paused/<string:pause_state>', methods=['POST'])
```

Reply:
@kaxil Why paused? It seems if we are asking it to do a thing that should be in present tense, no?